README.rst

```diff
@@ -1,27 +1,63 @@
 cardboardlint
 -------------
 
-Cheap lint solution for PRs
+Cheap lint solution for PRs.
 
-This is a wrapper for other linters, which only reports messages related to lines that have changed since some ancestor commit.
-It is a simple and more customizable alternative to tools like Hound CI, Stickler CI, Linthub.io etc.
+This is a wrapper for other linters, which only reports messages related to lines that
+have changed compared to some other commit, e.g. the head of the branch being merged into.
+It is a simple and more customizable alternative to tools like Hound CI, Stickler CI,
+Linthub.io etc.
 
 It can be used as follows:
 
-- Add a .cardboardlint.yml to the root of your source tree. It should at least contain the following:
+- Add a ``.cardboardlint.yml`` to the root of your source tree. It should at least contain the
+  following:
+
+  .. code:: yaml
 
-  .. code::
-  
       linters:
-          name_of_linter1:
-          name_of_linter2:
+          - name_of_linter1:
+              option_foo: bar
+          - name_of_linter2:
           ...
 
-- Download and run the linter to your CI tests (or locally)
+  A list of supported linters is easily deduced from the source code. Just look into
+  the ``cardboardlint/linter_*.py`` files. Each of these files has a module docstrong with
+  some explanations and a DEFAULT_CONFIG dictionary with the available configuration vars.
+
+  You can repeat any linter multiple times with different configuration settings. This can
+  be useful, e.g. when unit tests must be linted differently than the rest of the source
+  code.
```

Comment:
In this case, the ``.cardboardlint.yml`` will look something like this:

.. code:: yaml

     linters:
        - name_of_linter1:
            option_foo: bar
        - name_of_linter1:
            option_foo: gar